vuu-ui/packages/vuu-layout/test/layout-persistence/LocalLayoutPersistenceManager.test.ts

```diff
@@ -0,0 +1,535 @@
+import { Layout, LayoutMetadata } from "@finos/vuu-shell";
+import { afterEach, describe, expect, it, vi } from "vitest";
+import { LocalLayoutPersistenceManager } from "../../src/layout-persistence";
+import { LayoutJSON } from "../../src";
+import { getLocalEntity, saveLocalEntity } from "../../../vuu-filters/src/local-config";
```

Comment:
Apparently a few of these imports can be shortened

```suggestion
import { Layout, LayoutMetadata } from "@finos/vuu-shell";
import { afterEach, describe, expect, it, vi } from "vitest";
import { LayoutJSON, LocalLayoutPersistenceManager } from "../../src";
import { getLocalEntity, saveLocalEntity } from "@finos/vuu-filters";
```

Reply:
Do the tests still pass for you after making this change? I get a type error if I try that:
```
TypeError: registerComponent is not a function
```

I could change the `../../src` import for consistency.

Reply:
Ah yeah they don't. But happy with that consistency change 👍